spark/src/main/scala/org/apache/comet/expressions/CometCast.scala

```diff
@@ -70,9 +70,10 @@ object CometCast {
           case _ =>
             Unsupported
         }
-      case (_: DecimalType, _: DecimalType) =>
-        // https://github.com/apache/datafusion-comet/issues/375
-        Incompatible()
+      case (from: DecimalType, to: DecimalType) =>
```

Comment:
higher to lower precision conversion in Datafusion changing the integer part, hence marked it as incompatible().

Reply:
I will change the PR description to `part of` https://github.com/apache/datafusion-comet/issues/375 instead of `closing`...